cookiecutter.json

```diff
@@ -11,16 +11,16 @@
     "custom_domain": "",
     "top_level_domain_name": "com",
     "second_level_domain_name": "crowdbotics",
-    "project_package_name": "{{ cookiecutter.project_slug|lower|replace(' ', '')|replace('-', '') }}",
-    "database_option": ["h2", "mysql", "mongodb", "postgresql", "none"],
+    "project_package_name": "{{ cookiecutter.project_slug|lower|replace(' ', '')|replace('-', '')|replace('_', '') }}",
+    "database_option": ["mysql", "h2", "mongodb", "postgresql", "none"],
```

Comment:
Let's leave `h2` as the first (and default) choice.

Reply:
Done.